src/main/java/frc/robot/subsystems/shooter/commands/PrepareShooter.java

```diff
@@ -0,0 +1,33 @@
+package frc.robot.subsystems.shooter.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.shooter.Shooter;
+
+import java.util.function.DoubleSupplier;
+
+public class PrepareShooter extends CommandBase {
```

Comment:
A more appropriate name would be just `Shoot`

Reply:
But it's not exactly that. Shooting is when the ball is physically fed to the shooter.

Reply:
Shooting is when you outtake cargo through the shooter, i.e. when you shoot a ball